src/UI/Component/Input/Field/Factory.php

```diff
@@ -196,7 +196,8 @@ public function dependantGroup(array $inputs);
 	 *     1: >
 	 *      A checkbox MUST NOT be used whenever a user has to perform a binary choice where
 	 *      option is not automatically the inverse of the other (such as 'Order by Date' and
-	 *      'Order by Name'). A  Select Input or a Radio Group in MUST be used in this case.
+	 *      'Order by Name'). A TagInput Input, Select Input or a Radio Group in MUST be
```

Comment:
How does a Tag Input allow for binary choice? Binary choice between mutually exclusive but not inverse options must be done by selecting exactly one option. Is this a one-tag-only Input? 

One a more general approach. Why is this control discussed with form input elements? Is it not an input element that is doing its job very clearly outside a form context?

Reply:
@atoedt: That is exactly the point, it should not be used for binary choices due to this weird one-tag-control that would result out of that. Also not for other single-choice like tasks. I do not understand your comment concerning the form. It is proposed here as Input Field, may be used in different context, where form might be one of them. However, I would love to see it in many forms in future or even better, in filters. Note that the first use of this will be in a form (consult the linked FW-Entry).